src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__card card" data-qa="card">
+        <div class="card__img">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="APPLE A1419"
+          >
+        </div>
+
+        <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <p class="card__id">Код товара: 195434</p>
+        <div class="stars card__reviews">
+          <div class="stars--4 stars">
```

Comment:
Block stars inside block stars?

```suggestion
        <div class="card__reviews">
          <div class="stars--4 stars">
```